builders/build/eth-api/dev-env/hardhat.md

```diff
@@ -7,15 +7,21 @@ description: 了解如何使用Hardhat在Moonbeam上编译、部署和调试以
 
 ## 概览 {: #introduction }
 
-[Hardhat](https://hardhat.org/){target=_blank}是一个以太坊开发环境，可帮助开发人员管理和自动化构建智能合约和DApp所固有的重复性任务。Hardhat可以直接与Moonbeam的以太坊API交互，因此可以在部署智能合约至Moonbeam时使用。
+[Hardhat](https://hardhat.org/){target=_blank}是一个灵活可拓展的以太坊开发环境，它能帮助开发者简化智能合约开发过程。 由于Moonbeam与以太坊兼容，您可以在Moonbeam上使用Hardhat开发和部署智能合约。
```

Comment:
can you please update this entire page so all of the `target=_blank`s on this page are using `target=\_blank` instead?